From 6dce3b2e9974fb5c0c9540c3471d7277bc3f4ec4 Mon Sep 17 00:00:00 2001
From: Jose Dapena Paz <jdapena@igalia.com>
Date: Mon, 10 Jan 2022 19:25:53 +0000
Subject: IWYU: missing include for using memset in ScopedResState.
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix build error with libstdc++:
    ../../net/dns/public/scoped_res_state.cc: In constructor ‘net::ScopedResState::ScopedResState()’:
    ../../net/dns/public/scoped_res_state.cc:20:3: error: ‘memset’ was not declared in this scope
       20 |   memset(&res_, 0, sizeof(res_));
          |   ^~~~~~

Bug: 957519
Change-Id: Id6202959ebb15b7be25d966843ce25214b3407d3
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3377522
Reviewed-by: Eric Orth <ericorth@chromium.org>
Commit-Queue: José Dapena Paz <jdapena@igalia.com>
Cr-Commit-Position: refs/heads/main@{#957142}
---
 net/dns/public/scoped_res_state.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/dns/public/scoped_res_state.cc b/net/dns/public/scoped_res_state.cc
index f7dbb5e77cd86..9706d18b0f54e 100644
--- a/net/dns/public/scoped_res_state.cc
+++ b/net/dns/public/scoped_res_state.cc
@@ -4,6 +4,7 @@
 
 #include "net/dns/public/scoped_res_state.h"
 
+#include <cstring>
 #include <memory>
 
 #include "base/check.h"
-- 
2.32.0

